### PR TITLE
Include link and brief description to the projects board in the readme

### PR DESCRIPTION
**Pull request by bijington**
An initial PR to see if we can expose our 'plan' to the readers of the readme.

### CODE REVIEW

**Review by pictos — APPROVED** [review]

**Comment by VladislavAntonyuk** [comment]
What do you think if we first migrate to the new Projects?
12791661-0AD8-46ED-A071-C281F14E2072

the link may be changed

**Comment by brminnick** [comment]
> What do you think if we first migrate to the new Projects?

Smart. Makes sense to migrate to the new Project Board first.

**Comment by bijington** [comment]
I assumed we weren't migrating for a specific reason but never asked what it was 😅

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ All features are contributed by you, our amazing .NET community, and maintained
 
 And – the best part – the features you add to the .NET MAUI Toolkit may one day be included into the official .NET MAUI library! We leverage the Community Toolkits to debut new features and work closely with the .NET MAUI engineering team to nominate features for promotion.
 
+## Roadmap / plan
```
Comment (brminnick) [inline README.md:13]: Let's capitalize `Plan` to match the title case in the other titles

**Review by brminnick — COMMENTED** [review]
Thanks Shaun! 

I agree with Pedro - let's migrate to the new project board first. 

Things will be a bit busy/crazy this week thanks to the .NET 7 release, but I can prioritize the Project Board migration next week. 

We'll need to ensure it doesn't break msftbot 💯

File: README.md (hunk 1 above)
Comment (brminnick) [reply README.md:13]:
```suggestion
## Roadmap / Plan
```

**Comment by brminnick** [comment]
Hey Gang! I just tested MSFTBOT and confirmed it is not yet compatible with the new Project Board.

I'll go ahead and merge this for now, and once MSFTBOT is compatible, we'll migrate our Project Board to the new style.

**Review by brminnick — APPROVED** [review]

**Pull request merged**